app/controllers/checks_controller.rb

```diff
@@ -32,6 +33,10 @@ def assessment_code
     params[:assessment_code]
   end
 
+  def clear_early_result
+    session_data["early_result"]&.clear
```

Comment:
this should always exist so going to remove the nil guarding

Reply:
of course this actually doesn't always exist for checks that don't pass through income, doh!